scripts/reindex.py

```diff
@@ -58,6 +63,13 @@
                     ],
                     choices=config.catalogs,
                     help='The names of the catalogs to reindex.')
+parser.add_argument('--sources',
+                    default=config.reindex_sources,
+                    nargs='+',
+                    help='Limit remote reindexing to a subset of the configured sources. '
+                         'Supports shell-style * globbing to match multiple sources per argument. '
+                         'Each argument must match at least one configured source for every selected catalog. '
+                         'Default: %(default)s')
```

Comment:
Is this necessary?

Reply:
Not necessarily I suppose. Checking that it occurs in at least one catalog should be sufficient.

Reply:
I meant the inclusion of the default in the help string. The formatter should do that for us, no?

Reply:
Ah, you are correct/